code/modules/admin/topic.dm

```diff
@@ -1287,6 +1287,57 @@
 		NP.ckey = M.ckey
 		qdel(M)
 
+	else if(href_list["eraseflavortext"])
+		if(!check_rights(R_ADMIN))
+			return
+
+		var/mob/M = locateUID(href_list["eraseflavortext"])
+
+		if(!M.client)
```

Comment:
Best to also add a null check on `M`. It is an edge case but it can happen.

Reply:
Alright, I was copying the pattern as the other topics which don't have a null check, so I'll add it on the other ones as well

Reply:
Might have forgotten to push the changes. Else it's best to keep comments unresolved till they are actually resolved on the PR itself

Reply:
Aye, juggling multiple PRs at the same time is a recipe for a disaster, I'll get to it now. Thanks